RocketChat/Rocket.Chat.ReactNative
# Chore: Evaluate SearchMessagesView - TypeScript

## Proposed changes


## Issue(s)	


## How to test or reproduce

- Go to a channel
- Then, click on `Search` icon
- Check the messages from search message
- Also, test with an encryption channel 

## Screenshots

## Types of changes




## Checklist



## Further comments
